diagnostic.md

```diff
@@ -14,7 +14,7 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$('ol li, ul li');
```

Comment:
This is one selector that would select list items that are descendants of an ordered list, AND list items that are descendants of an unordered list.
These are the correct selectors:
$('ol > li');
$('ol').children();